### DISCUSSION TITLE
Show notification channels for all alerts

### DISCUSSION THREAD

**Discussion by weeco** [grafana/grafana#41366] [Alerting]
**What would you like to be added**:

In the alert rules overview (`/alerting/list`) I'd like to see all configured alert notification channels for each listed alert in the overview.

**Why is this needed**:

We have tens or hundreds of configured alerts in several organizations. We don't have any chance to figure out easily whether all the configured alert notification channels are correct or whether there are alerts which do not have any notification channel attached at all (!!!). 

Alerts which don't have a notification channel attached are not visibly highlighted or similar which makes it hard to detect "muted alerts".

**Solution:**

One could probably just add tags on each row which shows the names of the configured notification channels for the specific alert.

**Comment by rikimaru0345** [comment]
Hi,

seems easy to do! Just a few small changes here:
https://github.com/grafana/grafana/blob/master/public/app/features/alerting/AlertRuleItem.tsx

I would offer to contribute this in a PR! 😄 

A few questions before that though:
- Is there even a chance a PR for this would be accepted/merged?
- If so, under what conditions? Like for example, would I need to write additional tests?
- There is also the [angular module](https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/alertlist/module.html). Would I have to implement this in the angular as well? Or is enough to just change the react component?

**Comment by weeco** [comment]
@marefr Will you accept PRs for this? Could you answer @rikimaru0345 's questions?

**Comment by peterholmberg** [comment]
Sorry for the late reply to this. This is a great suggestion and I will move it to discussions as that is how we will handle feature requests moving forward.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
